wiki/Ranking_Criteria/en.md

```diff
@@ -15,6 +15,18 @@ Glossary
 - **Rules:** All rules are exactly that: **rules**. They are **not** guidelines and may **not** be broken under **any** circumstance.
 - **Guidelines:** Guidelines may be violated under **exceptional** circumstances. These exceptional circumstances must be warranted by an exhaustive explanation as of why the guideline has been violated and why not violating it will interfere with the overall quality of the creation.
 
+### Mapset
+
+- **Drain time:** The time during which the health-bar drains. Excludes break periods. Taiko mode’s drain time includes sliders and the intended length of spinners. 
+- **Reasonable Spread:** A mapset without drastically large differences between difficulties as dictated by difficulty-specific rules and guidelines. 
+- **Game-mode:** osu!, osu!taiko, osu!catch, and each of osu!mania’s key counts are considered individual game-modes.
+- **Single-mode mapset:** A mapset which contains difficulties of only one game-mode.
+- **Hybrid mapset:** A mapset which contains difficulties of multiple game-modes.
+- **Marathon:** A mapset which does not require a full spread of difficulties.
+- **Mapset host:** A user who uploads and manages a mapset.
+- **Guest difficulty:** A difficulty added to a mapset which is not created by a mapset host.
+- **Collaborative difficulty:** A single difficulty mapped by more than one mapper.
```

Comment:
Maybe mention that it's shortened to "collab"

Reply:
can do